ext/nnstreamer/tensor_filter/tensor_filter_pytorch_core.cc

```diff
@@ -209,14 +209,14 @@ TorchCore::validateOutputTensor (const at::Tensor output)
   auto tensor_shape = output.sizes ();
 
   if (tensor_shape[0] != 0 && outputTensorMeta.num_tensors != tensor_shape[0]) {
-    GST_ERROR ("Invalid output meta: different size");
+    g_error ("Invalid output meta: different size");
     return -1;
   }
 
   if (tensor_shape[0] == 0) {
     tensor_type otype = getTensorTypeFromTorch (output.scalar_type ());
     if (outputTensorMeta.info[0].type != otype) {
-      GST_ERROR ("Invalid output meta: different type");
+      g_error ("Invalid output meta: different type");
```

Comment:
how about using g_critical() ? 
G_LOG_LEVEL_ERROR makes fatal exception maybe. If it is ok, then ignore my comment. :)

Reply:
updated g_error() -> g_critical()